sdk/textanalytics/ai-text-analytics/karma.conf.js

```diff
@@ -24,28 +25,42 @@ module.exports = function(config) {
       "karma-env-preprocessor",
       "karma-coverage",
       "karma-remap-coverage",
-      "karma-junit-reporter"
+      "karma-junit-reporter",
+      "karma-json-to-file-reporter",
+      "karma-json-preprocessor"
     ],
 
     // list of files / patterns to load in the browser
-    files: ["dist-test/index.browser.js"],
+    files: ["dist-test/index.browser.js"].concat(
+      isPlaybackMode() ? ["recordings/browsers/**/*.json"] : []
+    ),
 
     // list of files / patterns to exclude
     exclude: [],
 
     // preprocess matching files before serving them to the browser
     // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors: {
-      "**/*.js": ["env"]
+      "**/*.js": ["env"],
+      "recordings/browsers/**/*.json": ["json"]
       // IMPORTANT: COMMENT following line if you want to debug in your browsers!!
       // Preprocess source file to calculate code coverage, however this will make source file unreadable
       // "test-browser/index.js": ["coverage"]
```

Comment:
should this not be commented by default?

Reply:
The comments make it sound like it isn't

Reply:
Per discussion we don't want to uncomment this because it will cause bad karma performance.